packages/wonder-blocks-modal/components/modal-launcher.md

```diff
@@ -14,9 +14,49 @@ const styles = StyleSheet.create({
     title: {
         marginBottom: 16,
     },
+
+    modalContent: {
+        margin: "0 auto",
+        marginTop: 40,
+        marginBottom: 40,
+        paddingLeft: 32,
+        paddingRight: 32,
+        maxWidth: 544,
+    },
 });
 
-const modal = ({closeModal}) => <TwoColumnModal
+const standardModal = ({closeModal}) => (
+    <StandardModal
+        title="Title"
+        content={
+            <View style={styles.modalContent}>
+                <Body tag="p">
+                    {
+                        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."
```

Comment:
I use string literals rather than inlining the string into the JSX, so that Prettier won't try to break it onto multiple lines. I think this is net better for readability, since the content of the string doesn't actually matter. What do you think?

Reply:
This is fine.  It might be cool if we had a `<LoremIpsum>` component so that we could see what things look like with placeholder text, but then the code example would be shorter.  Not necessary though.

Reply:
I was wondering about this. If I wanted to make some components just for examples, but _not_ publish them as part of the Wonder Blocks library we end up shipping in webapp, is there a good way to denote that?

Reply:
We could put them in a separate folder.  As long as we don't export them from `index.js` they won't end up in `webapp`.

Reply:
Oh cool! Do we tree-shake?

Reply:
Hmm... that's a good point.  Right now, update-wonder-blocks-shims.js just grabs all files (include test files too I think).  We'll want to update that script to exclude test files.  We could also set it up to exclude files in a certain directory.